### ISSUE TITLE
App not rendering VoiceOver J/shift+j commands in tests 1-4 of Checkbox Example (Two State)

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at-app#1660]
The test plan preview, test runner, and reports are all omiting the 'j' and 'shift+j' commands from tests 1-4 of the checkbox test plan even though they correctly show up in the built test plan outside of the aria-at.w3.org site.

In tests 1 through 4, 'j' and 'shift+j' are shown in this
[preview of the checkbox test plan in the master branch of the aria-at repo](https://aria-at.netlify.app/review/apg/checkbox.html).

In the following pages of the aria-at.w3.org app, the commands should be rendered, but they are not:
- [App test plan review page for Checkbox Example (Two State) V25.10.20](https://aria-at.w3.org/test-review/163663)
- [App runner view of test 3 for test plan V25.10.20](https://aria-at.netlify.app/tests/apg/checkbox/test-03-navForwardsToNotCheckedCheckbox-voiceover_macos.collected.html)
- [Report for test plan V25.10.20](https://aria-at.w3.org/report/163663/targets/515#result-NmQwMeyIxMiI6MTE4NH0zEwMj)
